projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -3920,32 +3943,56 @@ public function process_submission() {
 		 */
 		do_action( 'grunion_after_feedback_post_inserted', $post_id, $this->fields, $is_spam, $entry_values );
 
+		/**
+		 * Filter the title used in the response email.
+		 *
+		 * @module contact-form
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param string the title of the email
+		 */
+		$title   = apply_filters( 'jetpack_forms_response_email_title', __( 'You got a new response!', 'jetpack' ) );
 		$message = self::get_compiled_form_for_email( $post_id, $this );
 
-		array_push(
-			$message,
-			'<br />',
-			'<hr />',
-			__( 'Time:', 'jetpack' ) . ' ' . $time . '<br />',
-			__( 'IP Address:', 'jetpack' ) . ' ' . $comment_author_IP . '<br />', // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
-			__( 'Contact Form URL:', 'jetpack' ) . ' ' . $url . '<br />'
-		);
-
 		if ( is_user_logged_in() ) {
-			array_push(
-				$message,
-				sprintf(
-					// translators: the the name of the site.
-					'<p>' . __( 'Sent by a verified %s user.', 'jetpack' ) . '</p>',
-					isset( $GLOBALS['current_site']->site_name ) && $GLOBALS['current_site']->site_name ?
-						$GLOBALS['current_site']->site_name : '"' . get_option( 'blogname' ) . '"'
-				)
+			$sent_by_text = sprintf(
+				// translators: the the name of the site.
+				'<br />' . __( 'Sent by a verified %s user.', 'jetpack' ) . '<br />',
+				isset( $GLOBALS['current_site']->site_name ) && $GLOBALS['current_site']->site_name ? $GLOBALS['current_site']->site_name : '"' . get_option( 'blogname' ) . '"'
 			);
 		} else {
-			array_push( $message, '<p>' . __( 'Sent by an unverified visitor to your site.', 'jetpack' ) . '</p>' );
+			$sent_by_text = '<br />' . __( 'Sent by an unverified visitor to your site.', 'jetpack' ) . '<br />';
 		}
 
-		$message = join( '', $message );
+		$footer = join(
+			'',
+			/**
+			 * Filter the footer used in the response email.
+			 *
+			 * @module contact-form
+			 *
+			 * @since $$next-version$$
+			 *
+			 * @param array the lines of the footer, one line per array element.
+			 */
+			apply_filters(
+				'jetpack_forms_response_email_footer',
+				array(
+					'<br />',
+					'<hr />',
+					'<span style="font-size: 12px">',
+					__( 'Time:', 'jetpack' ) . ' ' . $time . '<br />',
+					__( 'IP Address:', 'jetpack' ) . ' ' . $comment_author_IP . '<br />', // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+					__( 'Source URL:', 'jetpack' ) . ' ' . $url . '<br />',
```

Comment:
I would recommend building those earlier, and making the values fully translatable with `sprintf`, so this will work in languages where the value may come before the the field info.